docs/content/commands/npm-unpublish.md

```diff
@@ -20,22 +20,30 @@ npm unpublish [<@scope>/]<pkg> --force
 
 ### Warning
 
-Consider using the `deprecate` command instead, if your intent is to encourage users to upgrade, or if you no longer want to maintain a package.
+Consider using the [deprecate](/commands/npm-deprecate) command instead,
+if your intent is to encourage users to upgrade, or if you no longer
+want to maintain a package.
 
 ### Description
 
-This removes a package version from the registry, deleting its
-entry and removing the tarball.
+This removes a package version from the registry, deleting its entry and
+removing the tarball.
 
-If no version is specified, or if all versions are removed then
-the root package entry is removed from the registry entirely.
+The npm registry will return an error if you are not [logged
+in](/commands/npm-login).
 
-Even if a package version is unpublished, that specific name and
-version combination can never be reused. In order to publish the
-package again, a new version number must be used. If you unpublish the entire package, you may not publish any new versions of that package until 24 hours have passed.
+If you do not specify a version or if you remove all of a package's
+versions then the registry will remove the root package entry entirely.
 
-To learn more about how unpublish is treated on the npm registry, see our <a href="https://www.npmjs.com/policies/unpublish" target="_blank" rel="noopener noreferrer"> unpublish policies</a>. 
+Even if you unpublish a package version, that specific name and version
+combination can never be reused. In order to publish the package again,
+you must use a new version number. If you unpublish the entire package,
+you may not publish any new versions of that package until 24 hours have
+passed.
 
+To learn more about how the npm registry treats unpublish, see our <a
```

Comment:
should we maybe put this higher up in the warning?

thinking we should make it front and center that people should not necessarily expect that their unpublish will actually work due to policies

Reply:
Moved it to the very very top